dhcpv6/option_prefixdelegation.go

```diff
@@ -12,75 +12,114 @@ type OptIAForPrefixDelegation struct {
 	iaId    [4]byte
 	t1      uint32
 	t2      uint32
-	options []byte
+	options []Option
```

Comment:
I would use capital letter to make this field public similar to OptIANA.

Reply:
Ah, like how I originally had it, of course. :)